tests/unit/disco/sagas/testDisco.js

```diff
@@ -0,0 +1,104 @@
+import SagaTester from 'redux-saga-tester';
+
+import { loadEntities } from 'core/actions';
+import { ErrorHandler } from 'core/errorHandler';
+import addonsReducer from 'core/reducers/addons';
+import apiReducer from 'core/reducers/api';
+import * as api from 'disco/api';
+import { getDiscoResults, loadDiscoResults } from 'disco/actions';
+import discoResultsReducer from 'disco/reducers/discoResults';
+import discoSaga from 'disco/sagas/disco';
+import { dispatchSignInActions } from 'tests/unit/amo/helpers';
+import {
+  createFetchDiscoveryResult, fakeDiscoAddon,
+} from 'tests/unit/disco/helpers';
+
+
+describe(__filename, () => {
```

Comment:
Is this still worth doing after #2928 was found to be okay?

I actually think this is a cute idea but I'd rather a helper did it universally–`describe(__filename, () => {` is sort of awkward to read.

If you want to leave it for now I guess that's okay though, no biggie... we aren't THAT strict about spec names and I think that's fine.